scenarios/steps/cp.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import os
+
+import sh
+import yaml
+from behave import *
+from assertpy import assert_that
+
+from qingstor.sdk.config import Config
+from qingstor.sdk.service.qingstor import QingStor
+
+config = Config().load_user_config()
+qingstor = QingStor(config)
+test_config_file_path = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), os.path.pardir)
+)
+with open(test_config_file_path + '/test_config.yaml') as f:
+    test_data = yaml.load(f)
+    f.close()
+bucket = qingstor.Bucket(test_data['bucket_name'], test_data['zone'])
+bucket.put()
+qsctl = sh.Command("qsctl")
+
+
+@given(u'a local file')
+def step_impl(context):
+    sh.mkdir("tmp")
+    sh.dd("if=/dev/zero", "of=tmp/中文测试", "bs=1048576", "count=1")
+    sh.dd("if=/dev/zero", "of=tmp/qsctl_test_big", "bs=1048576", "count=10")
+
+
+@when(u'copy to QingStor key')
+def step_impl(context):
+    command = qsctl(
+        "cp",
+        "tmp/{filename}".format(filename="中文测试"),
```

Comment:
Test data and expected behavior should be stored in feature files, such as object keys, file size, success status and error code.